test/clj/cider/nrepl/middleware/test_selector.clj

```diff
@@ -0,0 +1,29 @@
+(ns cider.nrepl.middleware.test-selector
```

Comment:
Do we need separate ns for the selector tests? Seems odd to me.

Reply:
happy to merge into `cider.nrepl.middleware.test-test`. The only reason to keep it separate that it is more integration like test eg. it needs `session/session-fixture` and sends `nREPL` messages while the `....test-test` ns is much simpler.

Reply:
Hmm, I didn't notice it had virtually no tests. That's odd. I guess it can certainly use a few. In general I think that most of the middleware ns tests should be integration tests anyways.